incubator/jaeger/templates/cassandra-schema-job.yaml

```diff
@@ -38,6 +38,30 @@ spec:
           value: {{ .Values.cassandra.config.dc_name | quote }}
         - name: CASSANDRA_PORT
           value: {{ .Values.storage.cassandra.port | quote }}
+        {{ if .Values.storage.cassandra.tls.enabled }}
+          - name: CASSANDRA_TLS
+            value: {{ .Values.storage.cassandra.tls.enabled }}
+          - name: CASSANDRA_TLS_SERVER_NAME
+            valueFrom:
+              secretKeyRef:
+                name: {{ .Values.storage.cassandra.tls.secretName }}
+                key: serverName
```

Comment:
nit: can we instead call the key `cn` in the tls secret?

Reply:
Why would it be better? According to the Jaeger docs, the env var name is CASSANDRA_TLS_SERVER_NAME.

Reply:
what about `commonName` then ? just cause that's the term used in a tls cert generally, but either way is fine :)

Reply:
Right.